wazo_confgend/generators/extensionsconf.py

```diff
@@ -253,26 +250,24 @@ def _extensions_features(self, conf, xfeatures, ast_writer):
             for exten_feature in cfeatures:
                 ast_writer.write_option('exten', exten_feature)
 
-        return options.getvalue()
-
-    def gen_dialplan_from_template(self, template, exten, output):
+    def gen_dialplan_from_template(self, template, exten, ast_writer):
         if 'priority' not in exten:
             exten['priority'] = 1
 
         for line in template:
-            prefix = 'exten =' if line.startswith('%%EXTEN%%') else 'same  =    '
+            prefix, padding = ('exten', '') if line.startswith('%%EXTEN%%') else ('same ', '    ')
 
             line = line.replace('%%CONTEXT%%', str(exten.get('context', '')))
             line = line.replace('%%EXTEN%%', str(exten.get('exten', '')))
             line = line.replace('%%PRIORITY%%', str(exten.get('priority', '')))
             line = line.replace('%%ACTION%%', str(exten.get('action', '')))
             line = line.replace('%%TENANT_UUID%%', str(exten.get('tenant_uuid', '')))
 
-            output.write(u'{} {}\n'.format(prefix, line))
-        output.write('\n')
+            ast_writer.write_option(prefix, '{}{}'.format(padding, line))
+        ast_writer.write_newline()
 
     def _generate_global_hints(self, output):
-        output.write(u'[usersharedlines]\n')
+        output.write(u'\n[usersharedlines]\n')
```

Comment:
Oh you missed understand me. The issue was not having newline **after** `usersharedline` section
ex:
```
[usersharedlines]                                                                                    
exten = 156521c1-ee1b-41cd-9137-e48507cfe53b,hint,Custom:156521c1-ee1b-41cd-9137-e48507cfe53b-mobile                                                                                      
[allaccess-key-2069-from-extern]                                                                     
exten = i,1,Playback(no-user-find)    
```

I would like to have the following result as before
```
[usersharedlines]                                                                                    
exten = 156521c1-ee1b-41cd-9137-e48507cfe53b,hint,Custom:156521c1-ee1b-41cd-9137-e48507cfe53b-mobile  
                                                                                    
[allaccess-key-2069-from-extern]                                                                     
exten = i,1,Playback(no-user-find)    
```

Reply:
Oh sorry. Ok, that makes more sense. I'll change it. Just need `output.write(u'[usersharedlines]\n\n')` then.